runelite-client/src/main/java/net/runelite/client/plugins/grounditems/GroundItemsOverlay.java

```diff
@@ -90,11 +92,6 @@ public GroundItemsOverlay(Client client, GroundItemsPlugin plugin, GroundItemsCo
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		if (!plugin.isHotKeyPressed() && config.itemHighlightMode() == MENU)
```

Comment:
So, when having bot overlay disabled and render tiles disabled, all items will be still looped through and checked if they are highlighted even when nothing happens? that seems wrong